src/pages/ItemList.js

```diff
@@ -140,9 +150,66 @@ function ItemList({ token }) {
           <button onClick={() => setSearchInput(() => '')}>Reset</button>
           {list.map((doc) => {
             const wasCheckedInLast24Hours =
+              // now - doc.last_purchased_date < ONE_SECOND;
               now - doc.last_purchased_date < ONE_DAY;
+
+            const inactiveItem =
+              msToDay(now - doc.last_purchased_date) >=
+              doc.previous_estimate * 2;
+
+            const itemStatus = () => {
```

Comment:
Really awesome solution to this problem!  It was really cool to see the aria-labels working with the screen reader!  The only suggestion I would make is moving the itemStatus function outside of the JSX and passing in the items it needs as arguments when it is called.      Like:        

 style={{
       backgroundColor: itemStatus(
                    doc.last_purchased_date,
                    doc.previous_estimate,
                    inactiveItem,
                  ).color,
                }}

Reply:
The functionality works, good job on this ticket, that was some logic to break down.

Reply:
> Really awesome solution to this problem! It was really cool to see the aria-labels working with the screen reader! The only suggestion I would make is moving the itemStatus function outside of the JSX and passing in the items it needs as arguments when it is called. Like:
> 
> style={{ backgroundColor: itemStatus( doc.last_purchased_date, doc.previous_estimate, inactiveItem, ).color, }}

Thank you! I knew there had to be a better way to write it, but my tired brain couldn't figure it out at the time.